werkzeug/wrappers.py

```
@@ -1443,12 +1484,43 @@ def make_conditional(self, request_or_environ):
             # wsgiref.
             if 'date' not in self.headers:
                 self.headers['Date'] = http_date()
+            if accept_ranges is True:
+                accept_ranges = "bytes"
+            elif accept_ranges is None:
+                accept_ranges = "none"
+            elif not isinstance(accept_ranges, string_types):
+                accept_ranges = None
```

We should raise a ValueError on invalid value of `accept_ranges` here.